content/kubermatic/master/advanced/_index.en.md

```diff
@@ -3,11 +3,8 @@ title = "Advanced"
 date = 2018-05-31T12:07:15+02:00
 weight = 25
 chapter = true
-pre = "<b>7. </b>"
 +++
 
-### Chapter 7
```

Comment:
Is this intended or it's some automated edit?

Reply:
I would remove all numbers